RHDEVDOCS-5390: Documenting Pipelines 1.10.4 RN (#61292)
openshift/openshift-docs: modules/op-release-notes-1-10.adoc

* `buckets`
* `kube-api-qps`
* `kube-api-burst`
* `threads-per-controller`

[id="release-notes-1-10-4_{context}"]
== Release notes for {pipelines-title} General Availability 1.10.4

With this update, {pipelines-title} General Availability (GA) 1.10.4 is available on {product-title} 4.11, 4.12, and 4.13.

[id="fixed-issues-1-10-4_{context}"]
=== Fixed issues

> * This update fixes the bundle conversion issue for `v1beta1` pipeline references. Now, the bundle conversion feature converts the `PipelineRef` bundle to the `Resolver` bundle. 

Reviewer: This update fixes the bundle resolver conversion issue for pipelineref field in PipelineRun. Now, the bundle conversion converts the `PipelineRef` bundle to the `Pipeline` kind in PipelineRun

Reviewer: This update fixes the bundle resolver conversion issue for pipelineref field in PipelineRun. Now, it sets the kind field to `Pipeline` after conversion

Reviewer 2: yeah it's was setting the kind field to `Task` before (even when resolving Pipeline).. which made thing act weird to not working in some cases.

Author: done

> * This update fixes the bundle resolver conversion issue for the `PipelineRef` field in a pipeline run. Now, the conversion feature sets the value of the `kind` field to `Pipeline` after conversion. 

Reviewer 3: This sentence is correct grammatically but I would suggest that you modify this sentence in the "before this update and after this update" template that you have used in the rest of the content to maintain uniformity.

> * Before this update, the `pipelinerun.timeouts` field was reset to `timeouts.pipeline` value, ignoring `timeouts.tasks` and `timeouts.finally` values. This update fixes the issue and sets the correct default timeout value for a `PipelineRun` resource.

Reviewer 4: ```suggestion
* Before this update, the `pipelinerun.timeouts` field was reset to the `timeouts.pipeline` value, ignoring the `timeouts.tasks` and `timeouts.finally` values. This update fixes the issue and sets the correct default timeout value for a `PipelineRun` resource.
```

> * Before this update, the controller logs were spammed with unnecessary data. This update fixes the issue.

Reviewer 4: Nitpick, to make it read more positive:
```suggestion
* Before this update, the controller logs contained unnecessary data. This update fixes the issue.
```
